packages/react-component-library/src/components/DataGrid/DataGrid.test.tsx

```diff
@@ -71,12 +77,6 @@ describe('DataGrid', () => {
     },
   ]
 
-  it('should set the `role` attribute to `grid`', () => {
```

Comment:
This test is redundant because it's implicit in the other uses of `screen.getByRole('grid')`.